xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxFFmpegArchive.cpp

```diff
@@ -0,0 +1,71 @@
+/*
+ *  Copyright (C) 2018 Arthur Liberman
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#include "DVDDemuxFFmpegArchive.h"
+
+#include "DVDDemuxUtils.h"
+#include "DVDInputStreams/DVDInputStream.h"
+#include "DVDInputStreams/DVDInputStreamFFmpegArchive.h"
+#include "cores/VideoPlayer/Interface/Addon/TimingConstants.h" // for DVD_NOPTS_VALUE
+#include "threads/SingleLock.h"
+#include "utils/log.h"
+
+CDVDDemuxFFmpegArchive::CDVDDemuxFFmpegArchive() :
+    m_bIsOpening(false), m_seekOffset(0)
+{ }
+
+bool CDVDDemuxFFmpegArchive::Open(std::shared_ptr<CDVDInputStream> pInput, bool streaminfo, bool fileinfo)
+{
+  m_bIsOpening = true;
+  bool ret = CDVDDemuxFFmpeg::Open(pInput, streaminfo, fileinfo);
+  m_bIsOpening = false;
+  return ret;
+}
+
+bool CDVDDemuxFFmpegArchive::SeekTime(double time, bool backwards, double *startpts)
```

Comment:
> Left-align `*` and `&` to the base type they modify.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#52-pointer-and-reference-types